src/packages/engine-core/src/__tests__/errors.test.ts

```diff
@@ -21,7 +21,7 @@ describe('getErrorMessageWithLink', () => {
 
       This is a nonrecoverable error which probably happens when the Prisma Query Engine has a panic.
 
-      https://github.com/prisma/prismaclientjs/issues/new?body=HiPrismaTeam%21MyPrismaClientjustcrashed.Thisisthereport%3A%0A%23%23Versions%0A%0A%7CName%7CVersion%7C%0A%7C%7C%7C%0A%7CNode%7CNODE_VERSION%7C%0A%7COS%7Cdarwin%7C%0A%7CPrismaClient%7C1.2.3%7C%0A%0A%23Description%0A%60%60%60%0AThisissomecrazydescription%0A%60%60%60%0A%0A%23%23Logs%0A%60%60%60%0Atestnamespacehello%0A%60%60%60&title=Thisisatitle&template=bug_report.md
+      https://github.com/prisma/prisma/issues/new?body=HiPrismaTeam%21MyPrismaClientjustcrashed.Thisisthereport%3A%0A%23%23Versions%0A%0A%7CName%7CVersion%7C%0A%7C%7C%7C%0A%7CNode%7CNODE_VERSION%7C%0A%7COS%7Cdarwin%7C%0A%7CPrismaClient%7C1.2.3%7C%0A%0A%23Description%0A%60%60%60%0AThisissomecrazydescription%0A%60%60%60%0A%0A%23%23Logs%0A%60%60%60%0Atestnamespacehello%0A%60%60%60&title=Thisisatitle&template=bug_report.md
```

Comment:
Something is weird with this test, all the spacing got removed, I guess from `.replace(/[\+-]/g, ''),` line 18

Reply:
<img width="498" alt="Screen Shot 2020-12-15 at 14 53 19" src="https://user-images.githubusercontent.com/1328733/102223961-a9ac3400-3ee5-11eb-8540-841da19df3c2.png">

Reply:
Yes, the goal of the test is not to have a perfect snapshot, but to have a snapshot that works across OS's.